src/Utilities/ToolTask.cs

```diff
@@ -714,7 +720,7 @@ private string GetTemporaryResponseFile(string responseFileCommands, out string
                 ExitCode = -1;
 
                 // Start the process
-                proc.Start();
+                StartTask(proc);
```

Comment:
StartToolProcess would be more descriptive. 
And it should be
proc = Start*(proc) as override will return a different process object.